pyrevitlib/pyrevit/revit/db/query.py

```diff
@@ -970,10 +970,17 @@ def get_connected_circuits(element, spare=False, space=False):
         circuit_types.append(DB.Electrical.CircuitType.Spare)
     if space:
         circuit_types.append(DB.Electrical.CircuitType.Space)
-
-    if element.MEPModel and element.MEPModel.ElectricalSystems:
-        return [x for x in element.MEPModel.ElectricalSystems
-                if x.CircuitType in circuit_types]
+    if circuit_types !=[DB.Electrical.CircuitType.Circuit]:
+        if HOST_APP.is_newer_than(2021, or_equal=True): # deprecation of ElectricalSystems in 2021
+            if element.MEPModel and element.MEPModel.GetElectricalSystems:
+                return [x for x in element.MEPModel.GetElectricalSystems
```

Comment:
`GetElectricalSystems` is a method. https://www.revitapidocs.com/2023/ef8e7119-9fa4-3024-d43d-591759b6098b.htm